routing/external_resources.rst

```diff
@@ -147,6 +147,65 @@ suppose you want to prefix all application routes with ``/site`` (e.g.
 The path of each route being loaded from the new routing resource will now
 be prefixed with the string ``/site``.
 
+.. note::
+
+    If any of the prefixed routes defines an empty path, Symfony adds a trailing
+    slash to it. In the previous example, an empty path prefixed with ``/site``
+    will result in the ``/site/`` URL. If you want to avoid this behavior, set
+    the ``trailing_slash_on_root`` option to ``false``:
+
+    .. configuration-block::
+
+        .. code-block:: php-annotations
+
+            use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+
+            /**
+             * @Route("/site", "trailing_slash_on_root"="false")
```

Comment:
this does not work on annotations: `trailing_slash_on_root` works only as an import option.